### ISSUE TITLE
ATEM Studio HD/ Stream Dec- Media Player Commands

### ISSUE THREAD

**Issue by B-Plummer** [bitfocus/companion-module-bmd-atem#81]
Hello All,
I am having an issue getting the stream deck to speak to the ATEM Studio HD over the Companion app Commands. Using the ATEM Instance Presets.

Everything else is working fine so far.. (Changing inputs/ color bars/ etc)
Any ideas?
I have deleted the buttons twice and no luck.
If I press the physical buttons on the ATEM, all works fine.

Thanks,
Brian P.

**Comment by Julusian** [comment]
Could you share a screenshot of the button config that isnt working for you?
Also could you confirm the version of companion you are running.

As far as Im aware everything should be working, but this info will help me be able to test it in my setup

**Comment by B-Plummer** [comment]
Hello All,
Attached are screen shots of the 15 button layout.
I have also attached a copy of my file for reference.
We are running bit focus companion V2.1.0
Please let me know what else I can get you to see this issue.
Thanks,


Best regards,
Brian Plummer

Brian@PlumPrecision.com<mailto:Brian@PlumPrecision.com>
www.PlumPrecision.com<http://www.PlumPrecision.com>
714-672-0030 Office
714-494-8059 Fax

[cid:227A469A-1C05-4627-B5A9-E4824646B42F]

**Comment by B-Plummer** [comment]
Hello All,
I think I figured it out. I needed to load the program MP1 & MP2 then the Preview MP1 & MP2… from there I can select what slide I want for each.
See attached screen shot.
So sorry for the confusion.
Thanks,



Best regards,
Brian Plummer

Brian@PlumPrecision.com<mailto:Brian@PlumPrecision.com>
www.PlumPrecision.com<http://www.PlumPrecision.com>
714-672-0030 Office
714-494-8059 Fax

[cid:227A469A-1C05-4627-B5A9-E4824646B42F]
